codesign/tool/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Copyright 2019 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Fetches corresponding dart sdk from CIPD for different platforms, builds
+# an executable binary of device_doctor to `build` folder.
+#
+# This currently supports linux and mac.
```

Comment:
Do we need to support linux for codesign tool?

Reply:
Let me update this